tests/unit/scripting.tcl

```diff
@@ -532,13 +527,39 @@ start_server {tags {"scripting"}} {
             {exec}
             {multi}
             {exec}
```

Comment:
let's add another command here as well (after the last SPOP),
this empty MULTI-EXEC is gonna be gone soon (see #9890), so in order to be sure the SPOP is not propagated, we need some INCR here.

Reply:
done